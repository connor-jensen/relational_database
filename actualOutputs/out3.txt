Schedule('M','113')? Yes(1)