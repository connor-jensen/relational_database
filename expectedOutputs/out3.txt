#checks if program will query on a predicate with no facts

  McKayl(X)? No